### DISCUSSION TITLE
in-game screen is partially cropped out when in skin editor mode

### DISCUSSION THREAD

**Discussion by Piripuu** [ppy/osu#23287] [Q&A] [answered]
So i am currently playing on the latest version of osu! lazer and i've never touched anything within skin editor (except for adding the pp gauge, thats all), and my legacyhealthdisplay is partially cropped out from the skin editor page.

![image](https://user-images.githubusercontent.com/128076921/233804489-9ce2fe87-eb25-40a1-b78b-f932114429f9.png) (initial placement)
![image](https://user-images.githubusercontent.com/128076921/233804616-ee81ade0-8b68-446e-a95d-108f66c613f9.png) (changed placement to fit the icon on the far right-hand side) 
![image](https://user-images.githubusercontent.com/128076921/233804673-1ca1741c-2977-4f7e-b73b-9ca69a414774.png) (how the in-game screen looks like with the changed placement, which you can tell something is wrong by the icon on the far right)

does anyone know why this is happening and if so, how to fix it?

**Comment by integerrr** [comment, marked as answer]
All skinning elements are placed relative to their anchors, you probably got the wrong anchor choice for those elements, you can right click those elements to check and change the anchors

**Reply by Piripuu** [reply]
hey integerrr, thanks for the reply! What i actually wanted to ask about was why does my LegacyHealthDisplay edit box protrude all the way into the grey, rectangle area at the right hand side of the screen?  

I dont know if its intended but all other skinning elements seem to fit nicely within this frame provided by the game, EXCEPT for the LegacyHealthDisplay.: 
![image](https://user-images.githubusercontent.com/128076921/233854951-f45c3f6d-c355-4161-a437-62094728d6b2.png)

As such, when im editing the length of the LegacyHealthDisplay, its quite hard to guage where exactly the icons are as it would protrude into this grey rectangle area at the right hand side of the editing screen: 
![image](https://user-images.githubusercontent.com/128076921/233855056-eac42285-3800-4342-86d2-1fd5eebffd50.png).

With regards to your tip about changing the anchor positions, that did help me better adjust the skin elements so thank you for that!

**Reply by integerrr** [reply]
That's because your `LegacyHealthDisplay` sprite is indeed that long, by the looks of it the creator of the skin has baked in a "cover" for the circular progress display, which would work in stable but will look all sorts of wrong in lazer.

**Reply by Piripuu** [reply]
ah i see, that would make sense. Thanks for the help!
